.buildkite/pipeline.yml

```diff
@@ -211,6 +211,14 @@ steps:
     trigger: "runtime-fe-end-to-end-tests"
     depends_on:
       - "build-chromium-linux-x86_64"
+    build:
+      env:
+        REPLAY_API_KEY: "rwk_o4eIfrkch6Y7eKFbJPGtKrVxNh0UiTF81KYU4IE3otr"
```

Comment:
Accidentally exposed `API_KEY`

Reply:
Not really, just a test API key to see if this will work. But point taken this should be in a secret